lib/html/pipeline/emoji_filter.rb

```diff
@@ -71,7 +71,42 @@ def asset_path(name)
 
       # Build an emoji image tag
       def emoji_image_tag(name)
-        "<img class='emoji' title=':#{name}:' alt=':#{name}:' src='#{emoji_url(name)}' height='20' width='20' align='absmiddle' />"
+        "<img #{img_html_attrs(name)}>"
```

Comment:
I'm wary of this change because unsafe user input could be injected here. Could you look whitelisting only attributes that make sense for a `img` element?